substrate/frame/merkle-mountain-range/Cargo.toml

```diff
@@ -16,15 +16,9 @@ targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
 codec = { workspace = true }
-frame-benchmarking = { optional = true, workspace = true }
-frame-support = { workspace = true }
-frame-system = { workspace = true }
+frame = { workspace = true, features = ["experimental", "runtime"] }
```

Comment:
what is the `experimental` feature for?

Reply:
I just follow other PRs. Yeah, it's a bit confusing, it won't compile after removing the feature.

```
polkadot-sdk git:(migrate-pallet-mmr) ✗ cargo check -p pallet-mmr
...
error[E0433]: failed to resolve: could not find `deps` in `frame`
  --> substrate/frame/merkle-mountain-range/src/lib.rs:65:2
   |
65 |     deps::sp_mmr_primitives::{
   |     ^^^^ could not find `deps` in `frame`
...
68 |     prelude::*,
   |     ^^^^^^^ could not find `prelude` in `frame`
...
```

Same concern in https://github.com/paritytech/polkadot-sdk/issues/6504#issuecomment-2568380349

Reply:
I think we should remove the `experimental` feature gate in frame crate:
```diff
diff --git a/substrate/frame/src/lib.rs b/substrate/frame/src/lib.rs
index f79a52bc6c5..44907a877d7 100644
--- a/substrate/frame/src/lib.rs
+++ b/substrate/frame/src/lib.rs
@@ -150,7 +150,6 @@
 //! * `runtime::apis` should expose all common runtime APIs that all FRAME-based runtimes need.
 
 #![cfg_attr(not(feature = "std"), no_std)]
-#![cfg(feature = "experimental")]
 
 #[doc(no_inline)]
 pub use frame_support::pallet;
```
Those PR is like making all pallets experimental.

I'll will put this message on the main issue tracker

Reply:
https://github.com/paritytech/polkadot-sdk/pull/7081/commits/1a41c6dbb1563d253952aa790f9dbae3fdc02c4f